cli/azd/test/functional/testdata/samples/containerapp/src/dotnet/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN dotnet publish -c Release -o out
 FROM mcr.microsoft.com/dotnet/aspnet:8.0
 WORKDIR /app
 COPY --from=build-env /app/out .
-EXPOSE 3100
-ENV ASPNETCORE_URLS=http://+:3100
+EXPOSE 8080
+ENV ASPNETCORE_URLS=http://+:8080
```

Comment:
Adjusting things in the Dockerfile so that the external port is the same as the one used by `dotnet publish` which allows us to re-use the IaC files between the `docker build` and `dotnet publish` flavors of the test.

Reply:
Nice! I think this also helped me orient myself around the default PORT change